tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = py27,py34
 skipsdist = True
 
 [testenv]
-recreate = True
+recreate = False
```

Comment:
Yes, the default tox environment should recreate the virtualenv each time, so this should be left as `recreate=True`. You can use `tox -e fast` if you don't want to recreate (but also skip most migrations).